pom.xml

```diff
@@ -302,4 +304,34 @@
             </plugin>
         </plugins>
     </reporting>
+    <profiles>
+        <!-- select the artefact version according to supported jdk --> 
+        <profile>
+            <id>downgrade-release-for-jdk8-it</id>
+            <activation>
+                <jdk>(,11]</jdk>
+            </activation>
+            <properties>
+                <netbeans.version.it>RELEASE126</netbeans.version.it>
+            </properties>
+        </profile>
+        <profile>
+            <id>downgrade-release-for-jdk11-it</id>
+            <activation>
+                <jdk>(11,17]</jdk>
+            </activation>
+            <properties>
+                <netbeans.version.it>RELEASE220</netbeans.version.it>
```

Comment:
wait, this should be 21 i think

Reply:
you have to watch out there. since the `]` doesn't do what it seems. `11]` will exclude 11.0.1.

* 8 -> NB 12.6
* 11 -> NB 21
* 17+ -> current

Reply:
hope this is right

Reply:
looks good i think. And it seems to work too ;)